launch.bat

```diff
@@ -66,10 +68,22 @@ goto Start
     :: Check for python installation
     where "%PYTHON_CMD%" >nul 2>&1
     if %errorlevel% neq 0 (
-        echo Python is not found.
+        echo %PYTHON_CMD% is not found.
+
+        if "%TEST_OTHER_PYTHON_PATH%"=="true" (
```

Comment:
I appreciate the configurability, but I think it's safe to just leave this as always a fallback. So you can remove this if clause and the variable `TEST_OTHER_PYTHON_PATH`.